frontend/src/scenes/insights/ActionFilter/entityFilterLogic.ts

```diff
@@ -45,7 +45,7 @@ export interface EntityFilterProps {
 // - filters
 // - setFilters
 // - typeKey
-export const entityFilterLogic = kea<entityFilterLogicType<BareEntity, EntityFilterProps, LocalFilter>>({
+export const entityFilterLogic = kea<entityFilterLogicType<BareEntity, LocalFilter>>({
```

Comment:
Context on this change? It does look like indeed `EntityFilterProps` is part of the deps

Reply:
Ah, I think I had an old version of `kea-typegen` that removed this – upgraded and its back